src/app/components/modules/TopRightMenu.jsx

```diff
@@ -0,0 +1,237 @@
+import React from 'react';
+import { Link } from 'react-router';
+import { connect } from 'react-redux';
+import { browserHistory } from 'react-router';
+import tt from 'counterpart';
+import DropdownMenu from 'app/components/elements/DropdownMenu';
+import Icon from 'app/components/elements/Icon';
+import * as userActions from 'app/redux/UserReducer';
+import * as appActions from 'app/redux/AppReducer';
+import Userpic from 'app/components/elements/Userpic';
+import VerticalMenu from 'app/components/elements/VerticalMenu';
+import LoadingIndicator from 'app/components/elements/LoadingIndicator';
+import NotifiCounter from 'app/components/elements/NotifiCounter';
+import { SIGNUP_URL } from 'shared/constants';
+
+const defaultNavigate = e => {
+    if (e.metaKey || e.ctrlKey) {
+        // prevent breaking anchor tags
+    } else {
+        e.preventDefault();
+    }
+    const a =
+        e.target.nodeName.toLowerCase() === 'a'
+            ? e.target
+            : e.target.parentNode;
+    browserHistory.push(a.pathname + a.search + a.hash);
+};
+
+function TopRightMenu({
+    username,
+    showLogin,
+    logout,
+    loggedIn,
+    vertical,
+    navigate,
+    toggleOffCanvasMenu,
```

Comment:
on steemitdev the menu toggle does not work -- when i click on the hamburger nothing happens